keylime/tenant.py

```diff
@@ -915,9 +930,9 @@ def do_cvstop(self):
             logger.error("Verifier at %s with Port %s timed out.", self.verifier_ip, self.verifier_port)
             sys.exit()
 
-        response_body = response.json()
```

Comment:
move line 920 `response_json = Tenant._jsonify_response(response, True)` to here...